index.html

```diff
@@ -18233,12 +18233,12 @@ <h2 id="requirements">
         Corporation); Mandyam, Giridhar (Qualcomm Innovation Center, Inc);
         Noble, Jer (Apple, Inc.); O'Callahan, Robert(Mozilla Foundation);
         Onumonu, Anthony (British Broadcasting Corporation); Paradis, Matthew
-        (British Broadcasting Corporation); Raman, T.V. (Google, Inc.);
-        Schepers, Doug (W3C/MIT); Shires, Glen (Google, Inc.); Smith, Michael
-        (W3C/Keio); Thereaux, Olivier (British Broadcasting Corporation); Toy,
-        Raymond (Google, Inc.); Verdie, Jean-Charles (MStar Semiconductor,
-        Inc.); Wilson, Chris (Google,Inc.) - Specification Co-editor; ZERGAOUI,
-        Mohamed (INNOVIMAX)
+        (British Broadcasting Corporation) - WG Chair; Raman, T.V. (Google,
```

Comment:
I'm happy to take this change.  Just not in this PR.

Reply:
Ha, its one of those things that was in the back of my head to do so thought i'd get it sorted. You are of course right though, will sort it.